tester_app/app/assets/stylesheets/_bootstrap-custom.scss

```diff
@@ -0,0 +1,55 @@
+// This file is used instead of
+// @import 'boostrap';
+// To allow for customization of what parts of bootstrap are imported.
+// Refer to this file: https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss
+
+// Core variables and mixins
+@import 'bootstrap/variables';
+@import 'bootstrap/mixins';
```

Comment:
@justin808 So then instead of `@import 'bootstrap-custom';` we can just `@import 'bootstrap'` and if someone wants to override the default mixins they can do just do that in the `_bootstrap-variables-customization.scss` file?